packages/utils-tests-helpers/src/hint-runner.ts

```diff
@@ -163,7 +163,7 @@ const validateResults = (t: ExecutionContext<HintRunnerContext>, sources: Map<st
     }
 
     if (results.length === 0) {
-        t.fail(`No results found, should be ${reports.length}`);
+        t.is(results.length, reports.length, `No results found, should be ${reports.length}`);
```

Comment:
I think this change becomes unnecessary if you remove the `reports` array in your new test cases.